packages/node/src/filesystem/DynamicFilesystem.js

```diff
@@ -0,0 +1,11 @@
+class DynamicFileSystem {
+  constructor(strategy) {
```

Comment:
Not sure if you were looking for feedback, if we are using classes and interfaces, this would be an excellent candidate for an interface for a strategy, there are some deviations below where loadChunk is async and not async - would help eliminate some of those small inconsistencies. Or types in general on those.

Reply:
Trying to avoid TS initially while I figure it out.